db/dcrpg/pgblockchain.go

```diff
@@ -3079,9 +3100,10 @@ func (pgb *ChainDB) storeTxns(msgBlock *MsgBlockPG, txTree int8,
 
 		// voteDbIDs, voteTxns, spentTicketHashes, ticketDbIDs, missDbIDs, err := ...
 		var missesHashIDs map[string]uint64
+		chainInfo := pgb.ChainInfo()
 		_, _, _, _, missesHashIDs, err = InsertVotes(pgb.db, dbTransactions, *TxDbIDs,
 			unspentTicketCache, msgBlock, pgb.dupChecks, updateExistingRecords,
-			pgb.chainParams, pgb.chainInfo)
+			pgb.chainParams, chainInfo)
```

Comment:
Part of why I like making the accessors like `ChainInfo()` is that they can go right here without new lines of code.  This is fine though.

Reply:
While making the other changes, please make this just `pgb.chainParams, pgb.ChainInfo())`